config/database.yml.example

```diff
@@ -10,6 +10,7 @@ test:
   encoding: utf8
   pool: 5
   timeout: 5000
+  strict: false
```

Comment:
Same as above

Reply:
This was added by @jywarren in the update process.

Reply:
Again, this was to get a test to pass. Thanks!